osgar/lib/pplanner.py

```diff
@@ -0,0 +1,54 @@
+"""
+Simple Path Planner
+"""
+
+
+def find_path(img, start, finish, verbose=False):
+    max_x, max_y = img.shape
+
+    if not (0 <= start[0] < max_x and 0 <=start[1] < max_y):
+        return None
+
+    if not img[start[1]][start[0]]:
+        return None
+
+    queue = [(start, None)]
+
+    expanded = set()
+    prev = {}
+
+
+    ret = None
+    while len(queue) > 0:
+        head, queue = queue[0], queue[1:]
```

Comment:
```suggestion
        (something, else), queue = queue[0], queue[1:]
```
Can we have a descriptive names for `head[0]` and `head[1]`?